core/collectors.py

```
@@ -124,3 +125,92 @@ def update_filters(
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.STDOUT,
+            )
+            out, _ = await proc.communicate()
+            text = out.decode(errors="replace")
+            match = self.LINE_RE.search(text)
+            if match:
+                cpu_pct = float(match.group("cpu"))
+                rss_kb = float(match.group("rss"))
+                rss_mb = rss_kb / 1024
+        except Exception:
+            pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)